versions/management/commands/import_reviews.py

```diff
@@ -0,0 +1,241 @@
+                            "Would set manager: "
+                            f"{first_name} {last_name} for {review.submission}"
+                        )
+                    else:
+                        user = _get_user_from_name(first_name, last_name)
+                        if user:
+                            review.review_manager = user
+                            review.save()
+
+    click.secho("\nDone!", fg="green")
+
+
+def _parse_table(table, past_results=False):
+    """Parse a review table and return review data"""
+    rows = table.find_all("tr")[1:]  # Skip header row
+    reviews = []
+
+    for row in rows:
+        cells = row.find_all("td")
+        if not cells or not cells[0].text.strip():
+            continue
+
+        review_data = {
+            "submission": cells[0].text.strip(),
+            "submitter_raw": cells[1].text.strip(),
+            "review_manager_raw": cells[2 if past_results else 3].text.strip(),
+            "review_dates": cells[3 if past_results else 4].text.strip(),
+            "github_link": "",
+            "documentation_link": "",
+        }
+
+        # Handle links for upcoming reviews
+        if not past_results:
+            links = cells[2].find_all("a")
+            for link in links:
+                if "github" in link.text.lower():
+                    review_data["github_link"] = link.get("href", "")
+                elif "documentation" in link.text.lower():
+                    review_data["documentation_link"] = link.get("href", "")
+
+        # Handle results for past reviews
+        results_data = []
+        if past_results:
+            result_cell = cells[4]
+
+            # First handle any linked results
+            for element in result_cell.contents:
+                if element.name == "del":
+                    link = element.find("a")
+                    if link:
+                        results_data.append(
+                            {
+                                "short_description": link.text.strip(),
+                                "announcement_link": link.get("href", ""),
+                                "is_most_recent": False,
+                            }
+                        )
+                elif element.name == "a":
+                    results_data.append(
+                        {
+                            "short_description": element.text.strip(),
+                            "announcement_link": element.get("href", ""),
+                            "is_most_recent": True,
+                        }
+                    )
+
+            # If no results were found, use the text content of the cell
+            if not results_data and (description := result_cell.text.strip()):
+                results_data.append({"short_description": description})
+
+        reviews.append((review_data, results_data))
+
+    return reviews
+
+
+def _parse_users_from_raw_names(raw_name_string: str) -> list[tuple[str, str]]:
```

Comment:
Are there any libraries that might handle this? 

I'm just reminded of https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/

Reply:
Fair question/reference, though for this narrow use case, this works fine and parses all names on the boost.org table "correctly". I've added a note in the docstring that this is definitely not for re-use 😊.